Library/Homebrew/keg_relocate.rb

```diff
@@ -97,7 +99,12 @@ def detect_cxx_stdlibs(_options = {})
   end
 
   def each_unique_file_matching(string)
-    Utils.popen_read("/usr/bin/fgrep", "-lr", string, to_s) do |io|
+    bsd = `/usr/bin/fgrep -V`.include?("BSD grep")
```

Comment:
Also I reckon BSD `grep` logic should live in https://github.com/Homebrew/brew/blob/master/Library/Homebrew/extend/os/mac/keg_relocate.rb or the converse in os/linux